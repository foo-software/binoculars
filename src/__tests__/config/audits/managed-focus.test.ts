import auditManagedFocus from '../../../config/audits/managed-focus';

describe('managed-focus audit', () => {
  it('should match snapshot', () => {
    expect(auditManagedFocus('en')).toMatchSnapshot();
  });
});
