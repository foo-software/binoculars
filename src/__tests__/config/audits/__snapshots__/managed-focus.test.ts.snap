// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`managed-focus audit should match snapshot 1`] = `
Object {
  "description": "If new content, such as a dialog, is added to the page, the user's focus is directed to it. [Learn more](https://web.dev/managed-focus/).",
  "title": "The user's focus is directed to new content added to the page",
}
`;
